Automattic/wp-calypso
# DIFM Lite: Redesigned In-progress screen

#### Changes proposed in this Pull Request

* Redesigned DIFM-L in-progress screen according to the new design: 2b998-pb/#plain

#### Testing instructions



* Log in and switch to a site that has the `difm-lite-in-progress` sticker. (It can be toggled from the Blog RC)
* Verify that the design matches the one in the link above.

![image](https://user-images.githubusercontent.com/5436027/152763172-578c1ee3-635e-40af-8564-7265f253e372.png)
